docs/en/changelog.md

```diff
@@ -1,6 +1,46 @@
 ## Changelog
 
 
+### V0.21 (1/29/2022)
+
+**Highlights**
+
+- Support Segmenter ([#955](https://github.com/open-mmlab/mmsegmentation/pull/955))
```

Comment:
Add a highlight for Officially Support CPUs training and inference

Reply:
Officially Support CPUs training and inference, please use the latest MMCV (1.4.4) to try it out.